codegen-server/src/main/kotlin/software/amazon/smithy/rust/codegen/server/smithy/customizations/AdditionalErrorsDecorator.kt

```diff
@@ -0,0 +1,87 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package software.amazon.smithy.rust.codegen.server.smithy.customizations
+
+import software.amazon.smithy.model.Model
+import software.amazon.smithy.model.shapes.MemberShape
+import software.amazon.smithy.model.shapes.OperationShape
+import software.amazon.smithy.model.shapes.ServiceShape
+import software.amazon.smithy.model.shapes.StructureShape
+import software.amazon.smithy.model.traits.ErrorTrait
+import software.amazon.smithy.model.traits.RequiredTrait
+import software.amazon.smithy.model.transform.ModelTransformer
+import software.amazon.smithy.rust.codegen.smithy.customize.RustCodegenDecorator
+
+/**
+ * Add at least one error to all operations in the model.
+ *
+ * When this decorator is applied, even operations that do not have a Smithy error attatched,
+ * will return `Result<OperationOutput, OperationError>`.
+ *
+ * To enable this decorator write its class name to a resource file like this:
+ * ```
+ * C="software.amazon.smithy.rust.codegen.server.smithy.customizations.AddInternalServerErrorToInfallibleOpsDecorator"
+ * F="software.amazon.smithy.rust.codegen.smithy.customize.RustCodegenDecorator"
+ * D="codegen-server/src/main/resources/META-INF/services"
+ * mkdir -p "$D" && echo "$C" > "$D/$F"
+ * ```
+ */
+class AddInternalServerErrorToInfallibleOpsDecorator : RustCodegenDecorator {
+    override val name: String = "FallibleOperations"
```

Comment:
Sorry for being a nitpick, but the name here should be the same as the class IMHO. Same for the other decorator..

Reply:
oh my bad, didn't notice, fixing